github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-periodics.yaml

```
@@ -31,7 +31,7 @@ periodics:
         ./pipeline-periodic.sh
       env:
       - name: GIMME_GO_VERSION
-        value: "1.19"
+        value: "1.21"
```

I think you need to specify the full release version
```suggestion
        value: "1.21.6"
```